src/AuthProvider.tsx

```diff
@@ -202,12 +202,15 @@ export const AuthProvider = (props: AuthProviderProps): JSX.Element => {
 
         void (async (): Promise<void> => {
             try {
+                let user = await userManager.getUser();
```

Comment:
- getUser might initiate events
- in `hasAuthParams()` we get the user twice

Therefor it might be better to do something like this:
```
            let user: User | void | null = null;
            try {
                // check if returning back from authority server
                if (hasAuthParams() && !skipSigninCallback) {
                    user = await userManager.signinCallback();
                    onSigninCallback && onSigninCallback(user);
                }
                user = !user ? await userManager.getUser() : user;
                dispatch({ type: "INITIALISED", user });
            } catch (error) {
                dispatch({ type: "ERROR", error: loginError(error) });
            }
```